content/articles/making-predictions-using-monte-carlo-simulation/index.md

```diff
@@ -0,0 +1,322 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /making-predictions-using-monte-carlo-simulation/
+title: Making Predictions Using Monte Carlo Simulation
+description: This tutorial discusses the monte carlo simulation process. The reader will understand the steps used in this process in order to get accurate results. 
+author: victor-kamau
+date: 2021-09-02T00:00:00-06:30
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/making-predictions-using-monte-carlo-simulation/hero.jpg
+    alt:  Making Predictions Using Monte Carlo Simulation Hero Image
+
+---
+
+With the growing technology especially in the field of AI(Artificial Intelligence), the knowledge of different methods of training a model is essential to programmers.
+
+<!--more-->
+
+These techniques help in automation of decision making process and help predict the probability of different outcomes when random variables are intercepted.
```

Comment:
Try to avoid single line paragraphs. Please join these up.

Reply:
I have corrected the paragraphs